src/lime/tools/HXProject.hx

```diff
@@ -26,6 +26,7 @@ class HXProject extends Script
 	public var app:ApplicationData;
 	public var architectures:Array<Architecture>;
 	public var assets:Array<Asset>;
+	public var assetsFolders:Array<String>;
```

Comment:
As I said in 2021, modifying `HXProject` isn't the way to do this. We'll need to revert all `HXProject` changes and try a different approach. [See my original comment for a detailed suggestion](https://github.com/openfl/lime/pull/1475#issuecomment-972369334).